### ISSUE TITLE
@google/model-viewer cannot be imported from basic create-react-app app

### ISSUE THREAD

**Issue by mikejernil** [GoogleWebComponents/model-viewer#538]
### Description

Importing @google/model-viewer in React & Adding the code and running causes this import error

#### Screenshot
![Screenshot 2019-05-08 at 12 20 37 PM](https://user-images.githubusercontent.com/43872457/57355620-0bde7800-718c-11e9-9690-1734c431f5af.png)

### Versions

- model-viewer: latest

**Comment by cdata** [comment]
@mikejernil it might be helpful if we could see your package.json and also get details about your bundler: are you using Webpack or something else? What does your config look like?

**Comment by senthil5053** [comment]
I'm also getting the exact same error when tried using it in my React Web app. Below is my package.json 
```
{
  "name": "Test",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "@google/model-viewer": "^0.3.1",
    "react": "^16.8.6",
    "react-bootstrap": "^1.0.0-beta.8",
    "react-dom": "^16.8.6",
    "react-router-dom": "^5.0.0",
    "react-scripts": "2.1.8"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
  "eslintConfig": {
    "extends": "react-app"
  },
  "browserslist": [
    ">0.2%",
    "not dead",
    "not ie <= 11",
    "not op_mini all"
  ]
}
```
And below is the React component in which I'm trying to use `model-viewer` : 
```
import React from 'react';
import '@google/model-viewer';

export default function TestComponent() {
return (
        <div>
<model-viewer src="images/wooden_chair.gltf"></model-viewer>
         </div>
} 
```
Could you please help on this ?
BTW, I don't have any other version of three.js in my project.

**Comment by cdata** [comment]
No worries @senthil5053 we are here to help.

The error you showed sounds like a bundler error to me. I will try to repro in a bit, since it could be something we can fix on our side (although it may not be).

**Comment by cdata** [comment]
@senthil5053 btw how are you initializing this project? Are you using `create-react-app` by any chance?

**Comment by mikejernil** [comment]
@cdata I'm using create-react-app , importing the js in script tag at index.html solved my problem. didn't try npm again

**Comment by mikejernil** [comment]
Also, i have another THREE npm installed at a parent directory

**Comment by senthil5053** [comment]
Yes @cdata, I did use `create-react-app` to create my react web app. And I'm using yarn to run it.
BTW, I could make it work by importing JS script tag in index.html.

**Comment by cdata** [comment]
Hey @senthil5053 and @mikejernil , I tested out importing `@google/model-viewer` built from master. I think the error you are seeing was fixed when we upgraded to Three.js r104 in #536 . These changes have not yet been released, but we expect to do a release in the next week or two.

If you have time, if either of you could confirm that building from master works for you, that would be helpful for us!

**Comment by cdata** [comment]
I'm going to mark this as fixed by #536 . Please ping this thread if you think it was not fixed in this change. Hopefully when we release v0.4.0 (which will be very soon), everything will work as expected.
